### ISSUE TITLE
Server Error - Bad Request - Key Expired

### ISSUE THREAD

**Issue by SArsalanK** [ram-on/SkyTube#759]
Server Error - Bad Request - Key Expired

![Screenshot_2020-04-20-21-55-48-794_free rm skytube extra](https://user-images.githubusercontent.com/2125947/79778392-19b3eb80-8352-11ea-8a7b-8cf34659f9fb.jpg)

**Comment by tingsu** [comment]
I also encouter this issue from time to time... a little annoying

**Comment by SArsalanK** [comment]
Any fix?

**Comment by okan35** [comment]
Just put your youtube api key or try with newpipe backend

**Comment by goetzc** [comment]
@okan35 I get the following error when entering a new YT API key and selecting `OK`:
```
An error occurred attempting to validate the API key you entered.
```
The app was installed from F-Droid. Is this a known issue or is there any know workaround or solution?

**Comment by jasonkhanlar** [comment]
Similar to this and maybe also similar to https://github.com/ram-on/SkyTube/issues/643

I just installed this app from F-Droid today, and immediately encountered several bugs/glitches, things not working, so now I'm testing and reporting a few things. I opened and closed the app repeatedly, and each time I see a tooltip appear like in the screenshots), with errors messages, a few different messages appear such as:

```
Server error:API key expired. Please renew the API key., reason: badRequest
```

or

```
Server error:The request cannot be completed because you have exceeded your <a href="/youtube/v3/getting-started#quota">quota</a>., reason: quotaExceeded
```

oh, actually, on the 10th time of starting the app, I actually see that it loaded some content!!!!

I wasn't expecting it to actually work ever! I just installed this app now, 9 times it failed with those two error messages, cycling between them randomly, but now, it actually loaded some video thumbnails!


but otherwise, i just opened it like 30 more times, and about 10% of the time it actually works and loads thumbnail content in the default Featured tab.
